.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+- Describe the proposed changes:
+
+  - Is there an example or test page to demonstrate any new or changed features?
+  - Does your PR include appropriate tests for source code alterations?
```

Comment:
- If you're adding or changing a public API, did you update the docs Usage sections of the README? If not, please provide a code snippet that demonstrates how to consume the new or updated API(s).

Reply:
Perfect.  On its way soon.

Reply:
Alright, review again please.